rmgpy/rmg/model.py

```diff
@@ -69,9 +69,9 @@ class Species(rmgpy.species.Species):
     solventStructure = None
     solventViscosity = None
     isSolvent = False # returns True if the species is the solvent and False if not
-    diffusionTemp = None
     isSolventinCoolProp = False # returns True if the solvent's data are available in CoolProp and False if not
     SolventNameinCoolProp = None # returns the solvent's name that can be recognized by CoolProp. If the solvent is unavailable in CoolProp, it returns None
+    rxnTemp = None # returns the reaction temperature if the liquidreactor is used.
```

Comment:
this variable seems to be very generic, as the SimpleReactor also has a fixed temperature. I wonder if there are other ways to retrieve that information.